docs/specs/aerogear-security-offline/index.md

```diff
@@ -1,5 +1,6 @@
 ---
-layout: basic
+layout: post
+section: docs
 title: Offline Storage Specification (OSS)
```

Comment:
My preliminary check `ack aerogear-security-offline` revealed there is really no link here, this spec  should be linked from Docs, right?